{{cookiecutter.repostory_name}}/envs/prod/docker-compose.yml

```diff
@@ -118,14 +118,57 @@ services:
       - backend-static:/srv/static:ro
       - ./media:/srv/media:ro
       - ./letsencrypt/etc:/etc/letsencrypt
+      - ./nginx/monitoring_certs:/etc/monitoring_certs
     links:
       - app:app
+      {% if cookiecutter.monitoring == 'y' %}
+      - cadvisor:cadvisor
+      - node-exporter:node-exporter
+      {% endif %}
     command: nginx -g 'daemon off;'
     ports:
       - 80:80
       - 443:443
+      {% if cookiecutter.monitoring == 'y' %}
+      - 10443:10443
+      {% endif %}
     logging:
       <<: *logging
 
+  {% if cookiecutter.monitoring == 'y' %}
+  node-exporter:
+    image: prom/node-exporter:latest
+    container_name: node-exporter
+    restart: unless-stopped
+    volumes:
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /:/rootfs:ro
+    command:
+      - '--path.procfs=/host/proc'
+      - '--path.rootfs=/rootfs'
+      - '--path.sysfs=/host/sys'
+      - '--collector.filesystem.mount-points-exclude=^/(sys|proc|dev|host|etc)($$|/)'
+    logging:
+      <<: *logging
+
+  cadvisor:
+    image: gcr.io/cadvisor/cadvisor:v0.40.0
+    container_name: cadvisor
+    privileged: true
```

Comment:
Isn't this sorta dangerous? I quickly googled and here they don't use `privileged` - https://prometheus.io/docs/guides/cadvisor/

Reply:
in some (several, actually) cases it's needed: https://github.com/google/cadvisor/blob/master/docs/running.md

but let's turn it off and see if it breaks somewhere :)